assets/sass/multipage.scss

```diff
@@ -78,4 +111,151 @@
       }
     }
   }
-}
\ No newline at end of file
+}
+
+#multipage .content {
+  // inside narrative, style usernames + forum titles
+  // excludes posts, DMs where <strong> is for usernames + headers
+  > p > strong {
+    @include type-sans;
+    font-size: .90rem;
+    letter-spacing: .2px;
+    line-height: 1;
+  }
+
+  // inside narrative, style pauses
+  hr {
```

Comment:
@celinenguyen instead of using these breaks, we have a shortcode to insert divider gifs. You can add them like this: `{{< anigif name="berries_on_stem" >}}`